src/Microsoft.OData.Edm/Vocabularies/CoreVocabularies.xml

```diff
@@ -310,4 +310,8 @@ Any simple identifier | Any type listed in `Validation.OpenPropertyTypeConstrain
     <Annotation Term="Core.LongDescription" String="The annotation value will usually be an expression, e.g. using properties of the binding parameter type for instance-dependent availability, or using properties of a singleton for global availability. The static value `null` means that availability cannot be determined upfront and is instead expressed as an operation advertisement." />
   </Term>
 
+  <TypeDefinition Name="LocalDateTime" UnderlyingType="Edm.String">
+    <Annotation Term="Core.Description" String="A string representing a Local Date-Time value with no offset." />

```

Comment:
Does http://odata.github.io/WebApi/#04-01-datetime-support need to be updated?

Reply:
That's still true. OData V4 spec doesn't include "DateTime" as built-in primitive type.

It's the next step to use the "LocalDateTime" type definition in Web API.

Reply:
exactly.  LocalDateTime is not an actual type; its type is string. The typedef puts constraints on the format of the string, and defines semantics around what the string represents.

---
In reply to: [229088414](https://github.com/OData/odata.net/pull/1308#discussion_r229088414) [](ancestors = 229088414)